PocketKit/Sources/PocketKit/Premium/PremiumUpgradeView.swift

```diff
@@ -175,10 +176,28 @@ Refunds are not available for unused portions of a subscription.
 }
 
 private struct PremiumTermsView: View {
+    @State var showSafari = false
+    @State var urlString = "https://getpocket.com/en/privacy/"
+    // TODO: URLs need to be localized
+    let privacyPolicyUrlString = "https://getpocket.com/en/privacy/"
+    let toSUrlString = "https://getpocket.com/en/tos/"
+
     var body: some View {
         HStack(spacing: 16) {
-            Button { } label: { Text("Privacy Policy").style(.terms) }
-            Button { } label: { Text("Terms of Service").style(.terms) }
+            Button(action: {
+                self.urlString = privacyPolicyUrlString
+                self.showSafari = true
+            }, label: { Text("Privacy Policy").style(.terms) })
+                .sheet(isPresented: $showSafari) {
+                    SafariView(url: URL(string: self.urlString)!)
```

Comment:
force unwrapping can be dangerous, should we check if this value is non optional first?